ref/general/mp-openapi.adoc

```diff
@@ -12,19 +12,29 @@
+OpenAPI, previously known as Swagger, defines a standard interface for documenting and exposing RESTful APIs.
+This standard interface enables both humans and computers to process a web service available through its REST API without directly accessing the source code of the web service.
+MicroProfile OpenAPI provides a set of Java interfaces and programming models that enable Java developers to natively produce OpenAPI documents from their JAX-RS applications.
 
-Developers can communicate with the REST API in their web service only if they have the relevant information to do so. It is important to document APIs when they are published so that other developers can use them.
+Developers can communicate with the REST API in their web service only if they have the relevant information to do so.
+It is important to document APIs when they are published so that other developers can use them.
 
-There are two main ways to generate a link:https://swagger.io/docs/specification/about/[OpenAPI] document. The developer can generate documentation of the REST API from the source code of the REST API itself. The reference document then produced, lists all the API endpoints and descriptions of how to use them. Alternatively, the developer can start by designing the API in the form of structured documentation that they can then augment with annotations in the code. The developer can even generate the stubs for the source code, including client libraries for the API, from this manually created documentation.
+There are two main ways to generate a link:https://swagger.io/docs/specification/about/[OpenAPI] document.
+The developer can generate documentation of the REST API from the source code of the REST API itself.
+The reference document then produced, lists all the API endpoints and descriptions of how to use them.
+Alternatively, the developer can start by designing the API in the form of structured documentation that they can then augment with annotations in the code.
+The developer can even generate the stubs for the source code, including client libraries for the API, from this manually created documentation.
 
 == Processing annotations
 
 You can augment the existing code with OpenAPI annotations, which provide detailed documentation for the REST API.
 
-For example, the following code example shows JAX-RS code to request the list of host names in the inventory. The code has been annotated with OpenAPI annotations to provide more useful and explanatory information about the operation:
+For example, the following code example shows JAX-RS code to request the list of host names in the inventory.
+The code has been annotated with OpenAPI annotations to provide more useful and explanatory information about the operation:
```

Comment:
The phrase "The code has been annotated" might sound a bit too passive. I think maybe change the entire sentence to "The OpenAPI annotations in the example provide more useful and explanatory information about the operation:"